### PR TITLE
added test for 10261

### PR DESCRIPTION
**Pull request by tjake**

### CODE REVIEW

**Comment by ptnapoleon** [comment]
@aboudreault Do you want to review this? And @tjake, can you fix the linter errors? `autopep8` will do it for you, if you `pip install autopep8 && autopep8 --in-place -a --ignore E501  materialized_views_test.py
`.

File: materialized_views_test.py [hunk 1]
```diff
@@ -638,6 +638,93 @@ def interrupt_build_process_test(self):
...
+        for event in trace.events:
+            desc = event.description
+            match = re.match(regex, desc)
+            if match:
+                if expect_digest:
+                    break
+                else:
+                    self.fail("Encountered digest mismatch when we shouldn't")
+        else:
+            if expect_digest:
```
Comment (aboudreault) [inline materialized_views_test.py]: Just to confirm that this **else** is intended to the **for** statement?

**Comment by aboudreault** [comment]
LGTM +1

File: materialized_views_test.py (hunk 1 above)
Comment (tjake) [inline materialized_views_test.py]: correct

**Comment by aboudreault** [comment]
waiting the patch to be committed before merging.

**Comment by mambocab** [comment]
CASSANDRA-10261 has been resolved, but @tjake can you rebase? Thanks.

**Comment by ptnapoleon** [comment]
Bump. Still need a rebase.

**Pull request merged**